src/esp32/entry.rs

```diff
@@ -50,16 +51,29 @@ pub fn serve_web(
 
             let mut client = builder.build().unwrap();
 
-            client.get_config().unwrap()
-        };
-
-        let robot = match repr {
-            RobotRepresentation::WithRobot(robot) => Arc::new(Mutex::new(robot)),
-            RobotRepresentation::WithRegistry(registry) => {
-                log::info!("building robot from config");
-                let r = LocalRobot::new_from_config_response(&cfg_response, registry).unwrap();
-                Arc::new(Mutex::new(r))
-            }
+            let (cfg_response, cfg_received_datetime) = client.get_config().unwrap();
+
+            let robot = match repr {
+                RobotRepresentation::WithRobot(robot) => Arc::new(Mutex::new(robot)),
+                RobotRepresentation::WithRegistry(registry) => {
+                    log::info!("building robot from config");
+                    let r = match LocalRobot::new_from_config_response(&cfg_response, registry) {
+                        Ok(robot) => {
+                            let logs = vec![config_log_entry(cfg_received_datetime, None)];
+                            client.push_logs(logs).expect("could not push logs to app");
+                            robot
+                        }
+                        Err(err) => {
+                            let logs = vec![config_log_entry(cfg_received_datetime, Some(&err))];
+                            client.push_logs(logs).expect("could not push logs to app");
+                            panic!("{}", err)
```

Comment:
nit: I don't like the panic here, we could ultimatey re-poll the config until the error is fixed. I think we can live with this for this pr but would add a ticket